framework/src/modules/chain/rounds/round.js

```diff
@@ -303,32 +303,26 @@ class Round {
 		if (exceptions.rounds[this.scope.round.toString()]) {
 			// Apply rewards factor
 			roundRewards.forEach((reward, subIndex) => {
-				roundRewards[subIndex] = new Bignum(reward.toPrecision(15))
-					.multipliedBy(
-						exceptions.rounds[this.scope.round.toString()].rewards_factor
-					)
-					.integerValue(Bignum.ROUND_FLOOR);
+				roundRewards[subIndex] = new BigNum(reward.toPrecision(15))
+					.times(exceptions.rounds[this.scope.round.toString()].rewards_factor)
+					.floor(BigNum.ROUND_FLOOR);
 			});
 
 			// Apply fees factor and bonus
-			roundFees = new Bignum(roundFees.toPrecision(15))
-				.multipliedBy(
-					exceptions.rounds[this.scope.round.toString()].fees_factor
-				)
+			roundFees = new BigNum(roundFees.toPrecision(15))
+				.times(exceptions.rounds[this.scope.round.toString()].fees_factor)
 				.plus(exceptions.rounds[this.scope.round.toString()].fees_bonus)
-				.integerValue(Bignum.ROUND_FLOOR);
+				.floor(BigNum.ROUND_FLOOR);
 		}
 
-		const fees = new Bignum(roundFees.toPrecision(15))
+		const fees = new BigNum(roundFees.toPrecision(15))
 			.dividedBy(ACTIVE_DELEGATES)
-			.integerValue(Bignum.ROUND_FLOOR);
-		const feesRemaining = new Bignum(roundFees.toPrecision(15)).minus(
-			fees.multipliedBy(ACTIVE_DELEGATES)
+			.floor(BigNum.ROUND_FLOOR);
```

Comment:
there are several places still uses input for `floor` function

Reply:
Found only in rounds in the code, will fix and push; did you found more in other files?

Reply:
yes there were several places 😉

Reply:
Changes pushed. Resolving

Reply:
@shuse2 check that many of the occurrences are `Math.floor()` which do take the number to be floored.